### ISSUE TITLE
Demo: Yellow guy interaction is wierd

### ISSUE THREAD

**Issue by Wavesonics** [jacopofar/godot-easy-dialogue#13]
Clicking right on him doesn't work, but slightly up and left does work? Some times?

I looked into it a little and don't see anything obviously wrong, so not sure whats going on.

**Comment by jacopofar** [comment]
Yeah, I saw it as well and have no idea why it happens. It seems to depend on the direction but I also couldn't find anything strange in the collision boxes.

Eventually I decided to ignore it since it's a problem with the demo more than the addon itself, but would be nice to understand what is going on.
